SDK/include/values.hpp

```diff
@@ -9,7 +9,11 @@ constexpr int CLASS_POOL_SIZE = 320;
 constexpr int OBJECT_POOL_SIZE = 2000;
 constexpr int OBJECT_POOL_SIZE_037 = 1000;
 constexpr int MAX_WEAPON_SLOTS = 13;
-constexpr int MAX_VEHICLE_MODELS = 611 - 400 + 1;
+constexpr int MIN_COMPONENT_ID = 1000;
+constexpr int MAX_COMPONENT_ID = 1193;
```

Comment:
It's not common for ranges in code to be inclusive like this.  A think a define for `1194` would be more idiomatic, and avoid some `+ 1`s (e.g. `MAX_COMPONENT_ID - MIN_COMPONENT_ID + 1`), but I'm not sure what the best name for it would be.

Reply:
I think this is unnecessary and still the final index of the interval comes in handy